doc/introduction/introduction.rst

```diff
@@ -1,162 +1,71 @@
-systems will excel at their own specific tasks.
+Users interested in applications of photonic quantum computers should check
+out the :doc:`applications` pages. Those wanting to dig deeper into the design
+of circuits can head to the :doc:`/introduction/circuits` page.
 
-Read more to get to grips with the photonic model of quantum computing and see how it
-contrasts with the qubit model.
+Developers can head to the :doc:`/development/development_guide` to see how
+they can contribute to Strawberry Fields.
 
-.. raw:: html
+Quickstart guides
+-----------------
 
-    <a href="intro_photonics/introduction.html">
-    <h5>read more <i class="fas fa-angle-double-right"></i></h5>
-    </a>
+In the following sections you can learn more about the key features of PennyLane:
 
-.. toctree::
-   :maxdepth: 2
-   :hidden:
-
-   intro_photonics/introduction
-
-Quantum algorithms
-##################
-
-
-.. customgalleryitem::
-    :tooltip: a photon
-    :description: javascript:void(0);
-    :figure: /_static/cloning.png
-    :size: 293 93
-
-Photonic quantum algorithms have been established for a number of protocols, including
-**teleportation**, **Hamiltonian simulation** and **quantum neural networks**. This section
-covers the technical details of these quantum algorithms.
-
-.. raw:: html
-
-    <a href="intro_photonics/quantum_algorithms.html">
-    <h5>read more <i class="fas fa-angle-double-right"></i></h5>
-    </a>
-
-.. toctree::
-   :maxdepth: 2
-   :hidden:
+.. image:: ../_static/code.png
+    :align: right
+    :width: 300px
+    :target: javascript:void(0);
 
-   intro_photonics/quantum_algorithms
+1. :doc:`circuits` shows how to construct and simulate photonic circuits in Strawberry Fields,
+   including applying quantum gates, performing measurements, and how to use our various
+   simulator backends.
 
-Conventions and formulas
-########################
+..
 
-.. customgalleryitem::
-    :tooltip: a photon
-    :description: javascript:void(0);
-    :figure: /_static/x_p.png
-    :size: 500 200
+2. :doc:`ops` outlines the various quantum photonic circuit building blocks
+   provided in Strawberry Fields  --- including gates, state preparations and measurements.
 
-Describing a quantum system requires a number of conventions to be fixed, including the
-value of Planck's constant :math:`\hbar` (we set :math:`\hbar=2` by default).
+..
 
-This section details formulas and constants used throughout Strawberry Fields and can be used for
-reference when constructing quantum circuits.
+4. :doc:`states` outlines the various statevector methods and attributes accessible
+   when using the built-in simulators. These provide useful information and processing
+   of the quantum state of the system.
 
-.. raw:: html
+..
 
-    <a href="intro_photonics/op_conventions.html">
-    <h5>read more <i class="fas fa-angle-double-right"></i></h5>
-    </a>
+5. :doc:`photonic_hardware` gives an overview of building, submitting, and managing

```

Comment:
This could be the first one, since that's how it appears in the table of contents

Reply:
Oh, I was thinking of moving the hardware quickstart to 5th in the TOC, partly because I was looking at it, and realised it had assumed knowledge on circuits and operations etc. So it makes a bit more sense for the other quickstarts to come first.